### DISCUSSION TITLE
Option to enable receiving and/or viewing emails in plain text

### DISCUSSION THREAD

**Discussion by d8f6** [tutao/tutanota#6272] [Ideas]
I know that there is an option to send emails in plain text, but I would also like to receive and/or view them in plain text as opposed to html. I do not use any html features, and the change of color (from dark to light theme) when viewing them throws me off. Okay, thanks.

This idea/request was inspired by this issue I and some other users had on the android mobile app: https://github.com/tutao/tutanota/issues/6250, but is relevant for all user apps.

**Comment by charlag** [comment]
There is already an issue for that:
https://github.com/tutao/tutanota/issues/1588

**Comment by d8f6** [comment]
@charlag 
I see, I did not scroll all the way down the thread to see that this was discussed there. I will close this as a duplicate (although I am specifically referring to incoming emails as opposed to composition).
